mindsdb/integrations/handlers/langchain_handler/tools.py

```diff
@@ -157,26 +158,48 @@ def _get_rag_params(pred_args: Dict) -> Dict:
     return rag_params
 
 
-def _build_retrieval_tool(tool: dict, pred_args: dict):
+def _build_retrieval_tool(tool: dict, pred_args: dict, skill: db.Skills):
     """
     Builds a retrieval tool i.e RAG
     """
     # build RAG config
 
     tools_config = tool['config']
 
-    mindsdb_path = pred_args['mindsdb_path']
 
     # we update the config with the pred_args to allow for custom config
     tools_config.update(pred_args)
 
     rag_params = _get_rag_params(tools_config)
 
     if 'vector_store_config' not in rag_params:
-
-        rag_params['vector_store_config'] = {'persist_directory': mindsdb_path('persisted_chroma')}
-
-    rag_config = RAGPipelineModel(**rag_params)
```

Comment:
what errors do you run into? 

Perhaps we could fix by pinning pydantic version in langchain_handler requirements? (fine for now though)